eve/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+version: 0.2
+
+branches:
+  default:
+    stage: pre-merge
+
+stages:
+  pre-merge:
+    worker: &master-worker
+      type: docker
+      path: eve/workers/master
+      volumes:
+        - '/home/eve/workspace'
+    steps:
+      - Git:
+          name: fetch source
+          repourl: '%(prop:git_reference)s'
+          shallow: True
+          retryFetch: True
+          haltOnFailure: True
+      - ShellCommand:
+          name: install dependencies
+          command: npm install
+      - ShellCommand:
+          name: run lint yml
+          command: npm run --silent lint_yml
+      - ShellCommand:
+          name: run lint
+          command: npm run --silent lint -- --max-warnings 0
+      - ShellCommand:
+          name: run lint_md
+          command: npm run --silent lint_md
+      - ShellCommand:
+          name: run test
+          command: npm run --silent test
+      - ShellCommand:
+          name: run ft_test
+          command: npm run ft_test
+      - ShellCommand:
+          name: run executables tests
+          command: npm install && npm test
```

Comment:
Why an other npm install ?
Should we change the order ? like first `npm run --silent test` and then `npm run ft_test`

Reply:
Actually the `npm install && npm test`  was defined in the `master` version of the `circle.yml`

Indeed it's not available in the 6.4 version so I don't have any good explanation. But it works so I've decided to leave it. (we can remove it in this version if you think it doesn't make sense here).

If you take a look at the master version, there's the following line:
```shell
cd lib/executables/pensieveCreds && npm install && npm test
```
Which correspond to this step.

Reply:
MHh this actually appeared much later than 6.4, and should be added afterwards on later versions/branches.

Reply:
As discuss offline. Let's remove it here, forward this change and create and other PR targeting rel/7.2 to add it where it make sens . 
Thanks :)